test/hotspot/jtreg/compiler/vectorapi/AllBitsSetVectorMatchRuleTest.java

```diff
@@ -42,8 +42,8 @@
  * @key randomness
  * @library /test/lib /
  * @requires vm.compiler2.enabled
- * @requires vm.cpu.features ~= ".*asimd.*"
- * @summary AArch64: [vector] Make all bits set vector sharable for match rules
+ * @requires (os.simpleArch == "aarch64" & vm.cpu.features ~= ".*asimd.*") | (os.simpleArch == "riscv64" & vm.cpu.features ~= ".*zvbb.*")
+ * @summary [vector] Make all bits set vector sharable for match rules
```

Comment:
Better to keep the original summary (`AArch64: [vector] Make all bits set vector sharable for match rules`) which maps to https://bugs.openjdk.org/browse/JDK-8287984